src/data/showcases.js

```diff
@@ -1198,6 +1198,14 @@ export const Showcases = [
     source: null,
     tags: ["wallet", "nftsupport"],
   },
+  {
+    title: "NOWNodes",
+    description: "Get instant access to Cardano RPC nodes, indexer, and WebSocket connection. Start seamless integration on a FREE tariff plan. You can focus on growing your web3 products with us.",
```

Comment:
The marketing language here is unacceptable.  You can compare with other Dev Portal listings to rewrite this with the correct tone: to simply _include lists of features_ (emphasising any _distinctive_ features) without incentives or coercion.  If you need help doing this please post here and I'll make some recommendations.

Reply:
thank you @AnnaNodes for https://github.com/cardano-foundation/developer-portal/pull/1149/commits/9193fc57a9864a5110d710fcef55843683dc9a93 - the style is very appropriate now.